tools/scripts/requirements.txt

```diff
@@ -5,7 +5,7 @@ numpy
 # NOTE: You can also build this dependency from source, per ./notebooks/README.md.
 # NOTE: The builder's version of tiledbsoma MUST be <= the API's tiledbsoma version, to ensure reader compatibility
 # of TileDB on-disk storage format
-tiledbsoma==0.5.0a6
+tiledbsoma==0.5.0a10
```

Comment:
a11 is out now (and I've released cell_census on top of it)

Reply:
yup, will upgrade; just wanted to get things working on a11 before introducing any other problems.